### PR TITLE
Pioneer 15 deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated the following deploy csvs in preparation  for Pioneer 15 cruise:
CP01CNPM-00004
CP01CNSM-00013
CP02PMUO-00016
CP03ISPM-00004
CP03ISSM-00012
CP04OSSM-00012

All CUIDs from previous cruises were also updated in the Surface Mooring deploy csvs.  Those are the only changes to the previous deployments even though it appears that the entire deploy csvs are highlighted - at least in my view).

### CODE REVIEW

File: deployment/CP01CNPM_Deploy.csv [hunk 1]
```diff
@@ -26,3 +26,12 @@ AR39,,AR44,,CP01CNPM-WFP01-02-DOFSTK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23
...
+AR48,,,,CP01CNPM-RII01-02-ADCPTG010,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGINS-ADCPTG-24544,40,-70,,,,
+AR48,,,,CP01CNPM-SBS01-00-STCENG000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGCON-BSTC01-00013,40,-70,,0,,
+AR48,,,,CP01CNPM-SBS01-01-MOPAK0000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGINS-MOPAK0-11360,40,-70,,0,,
+AR48,,,,CP01CNPM-WFP01-00-WFPENG000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-01-VEL3DK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-02-DOFSTK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-03-CTDPFK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-04-FLORTK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-05-PARADK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,40,-70,,,,
```
Comment (cdobs) [inline deployment/CP01CNPM_Deploy.csv:37]: Looks good. Confirmed SNs with config sheet

File: deployment/CP03ISPM_Deploy.csv [hunk 2]
```diff
@@ -26,3 +26,12 @@ AR39,,AR44,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24
...
+AR48,,,,CP03ISPM-RII01-02-ADCPTG010,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGINS-ADCPTG-20496,40,-70,,,,
+AR48,,,,CP03ISPM-SBS01-00-STCENG000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGCON-BSTC01-00018,40,-70,,0,,
+AR48,,,,CP03ISPM-SBS01-01-MOPAK0000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGINS-MOPAK0-58481,40,-70,,0,,
+AR48,,,,CP03ISPM-WFP01-00-WFPENG000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-01-VEL3DK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-03-CTDPFK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-04-FLORTK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-05-PARADK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40,-70,,,,
```
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv:37]: Looks good. Confirmed SNs with config sheet

File: deployment/CP02PMUO_Deploy.csv [hunk 3]
```diff
@@ -134,3 +134,12 @@ AR44,,,,CP02PMUO-WFP01-02-DOFSTK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015
...
+AR48,,,,CP02PMUO-RII01-02-ADCPSL010,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGINS-ADCPSL-22846,39,-70,,,,
+AR48,,,,CP02PMUO-SBS01-00-STCENG000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGCON-BSTC01-00005,39,-70,,0,,
+AR48,,,,CP02PMUO-SBS01-01-MOPAK0000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGINS-MOPAK0-11358,39,-70,,0,,
+AR48,,,,CP02PMUO-WFP01-00-WFPENG000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-01-VEL3DK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-02-DOFSTK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-03-CTDPFK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-04-FLORTK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-05-PARADK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39,-70,,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:145]: Looks good. Confirmed SNs with config sheet

File: deployment/CP01CNSM_Deploy.csv [hunk 4]
```diff
@@ -1,409 +1,443 @@
...
+AR48,,,,CP01CNSM-SBD11-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-BDCL11-00009,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-01-MOPAK0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-MOPAK0-43264,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-02-HYDGN0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-HYDGN0-11407,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD11-06-METBKA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-METLGR-00003,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-00-DCLENG000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGCON-BDCL12-00009,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-03-HYDGN0000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-HYDGN0-49860,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-04-PCO2AA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-PCO2AA-33183,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-05-WAVSSA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-WAVSSA-06271,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-06-METBKA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-METLGR-00007,40,-70,,0,,
+AR48,,,,CP01CNSM-SBD12-08-FDCHPA000,13,1,2020-10-28T00:00:00,,CGMCP-01CNSM-00013,,CGINS-FDCHPA-75008,40,-70,,-5,,
```
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv:443]: Looks good. Compared serial numbers to config sheet and confirmed CUID changes

File: deployment/CP03ISSM_Deploy.csv [hunk 5]
```diff
@@ -1,343 +1,374 @@
...
+AR48,,,,CP03ISSM-RID27-03-CTDBPC000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-CTDBPC-50002,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-DOSTAD-00387,40,-70,,7,,
+AR48,,,,CP03ISSM-SBC11-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BCPM01-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BDCL11-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-01-MOPAK0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-MOPAK0-11839,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-02-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-41692,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-06-METBKA000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-METLGR-00033,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BDCL12-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-03-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-49868,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-04-PCO2AA000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-PCO2AA-33182,40,-70,,0,,
```
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:374]: Looks good. Reviewed CUIDs and serial numbers with config sheet

File: deployment/CP04OSSM_Deploy.csv [hunk 6]
```diff
@@ -1,342 +1,373 @@
...
+AR48,,,,CP04OSSM-RID27-03-CTDBPC000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-CTDBPC-50003,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00391,39,-70,,7,,
+AR48,,,,CP04OSSM-SBC11-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BCPM01-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BDCL11-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-01-MOPAK0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-MOPAK0-44005,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-02-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-46156,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-06-METBKA000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-METLGR-00038,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BDCL12-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-03-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-41552,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-04-PCO2AA000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PCO2AA-35260,39,-70,,0,,
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:373]: Looks good. Reviewed CUIDs and compared SNs with config sheet

File: deployment/CP02PMUO_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:145]: Looks good. Confirmed SNs with config sheet

File: deployment/CP03ISPM_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv:37]: Looks good. Confirmed SNs with config sheet

**Review by cdobs — APPROVED** [review]
Everything reviewed and looks good to me. Approved pending swap of CP04OSSM PCO2W to spare (SN C0098). If that swap becomes permanent then I will review once the deploy csv is updated

File: deployment/CP04OSSM_Deploy.csv [hunk 7]
```diff
@@ -1,342 +1,373 @@
...
+AR39,,,,CP04OSSM-SBD11-06-METBKA000,11,1,2019-09-29T14:46:00,,CGMCP-04OSSM-00011,,CGINS-METLGR-00037,39.9368,-70.8865,,0,447,
+AR39,,,,CP04OSSM-SBD12-00-DCLENG000,11,1,2019-09-29T14:46:00,,CGMCP-04OSSM-00011,,CGCON-BDCL12-20003,39.9368,-70.8865,,0,447,
+AR39,,,,CP04OSSM-SBD12-03-HYDGN0000,11,1,2019-09-29T14:46:00,,CGMCP-04OSSM-00011,,CGINS-HYDGN0-49857,39.9368,-70.8865,,0,447,
+AR39,,,,CP04OSSM-SBD12-04-PCO2AA000,11,1,2019-09-29T14:46:00,,CGMCP-04OSSM-00011,,CGINS-PCO2AA-35264,39.9368,-70.8865,,0,447,
+AR48,,,,CP04OSSM-MFC31-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-MCPM03-00013,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-MDCL35-00050,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-01-ADCPSJ000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-ADCPSJ-22642,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-02-PRESFC000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PRESFC-01400,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-04-VELPTB000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-VELPTB-11986,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-05-PCO2WB000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PCO2WB-C0098,39,-70,,,,
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:348]: swap from C0097 to C0098 looks good

**Review by cdobs — APPROVED** [review]
swap from C0097 to C0098 looks good

File: deployment/CP01CNPM_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP01CNPM_Deploy.csv:37]: Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00127, and WFP instrument cal docs.

File: deployment/CP03ISPM_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP03ISPM_Deploy.csv:37]: Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00126, and WFP instrument cal docs.

File: deployment/CP02PMUO_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/CP02PMUO_Deploy.csv:145]: Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00127, and WFP instrument cal docs.

File: deployment/CP01CNSM_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/CP01CNSM_Deploy.csv:443]: Looks good.

File: deployment/CP03ISSM_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/CP03ISSM_Deploy.csv:374]: Looks good.

File: deployment/CP04OSSM_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/CP04OSSM_Deploy.csv:373]: Looks good.

**Review by snwhite — APPROVED** [review]
All files look good including OSSM PHSEN swap.

**Pull request merged**